cinema/views.py

```diff
@@ -12,6 +14,8 @@
     MovieDetailSerializer,
     MovieSessionDetailSerializer,
     MovieListSerializer,
+    OrderSerializer,
+    OrderListSerializer,
 )
```

Comment:
Code Style issue: The imports are not grouped using parentheses `()`. This should be corrected to improve readability.